src/lib/utilities/Button.svelte

```diff
@@ -4,7 +4,7 @@
     export let onClick;
     export let id;
 </script>
-<button class:minimal class:animated={!minimal} {type}  {id} on:click={onClick}>
+<button class:minimal class:animated={!minimal} type={type} id={id} on:click={onClick}>
```

Comment:
Tbh I don't see why this was changed? I think the previous way was more concise (imo) so I'll revert to that unless it breaks stuff